### PR TITLE
Client config contact #13637 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
~***DO NOT MERGE. DOCS REPO IS FROZEN***~

# Description

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [ ] Manager-4.2
- [ ] Manager-4.1

# Links

Fixes https://github.com/SUSE/spacewalk/issues/13637

### CODE REVIEW

File: modules/client-configuration/pages/contact-methods-traditional.adoc [hunk 1]
```diff
@@ -3,11 +3,42 @@
...
+Push with Tunnel
+////
 Push via SSH is used in environments where clients cannot reach the {productname} Server directly.
 In this environment, clients are located in a firewall-protected zone called a DMZ.
 No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
 
-OSAD is an alternative contact method between {productname} and traditional clients.
-OSAD allows traditional clients to execute scheduled actions immediately.
+Traditional SSH Push with Tunnel::
+is the same as SSH Push, but additionally tunnels HTTP/HTTPS traffic (for package download) via SSH.
```
Comment (Loquacity) [inline modules/client-configuration/pages/contact-methods-traditional.adoc:44]:
```suggestion
The same as SSH Push, but tunnels HTTP/HTTPS traffic (for package download) via SSH.
```

**Comment by 0rnela** [comment]
@keichwa -- what is the up-to-date status od this PR from yoyr pow?

**Comment by keichwa** [comment]
I'll create a new PR.  But please, let this stay open. I need it for inspiration.

**Comment by keichwa** [comment]
Now superseded by https://github.com/uyuni-project/uyuni-docs/pull/1860/ .

**Pull request closed without merging**